bescort.lic

```diff
@@ -172,6 +178,7 @@ class Bescort
   def search
     fput('search')
     pause
```

Comment:
Instead of calling `pause` twice, you can pass it an arg (of seconds), e.g. `pause 1`.